source/includes/inkling-reference/_schema.html.md

```diff
@@ -261,3 +266,4 @@ Select the Inkling tab to view some examples of valid and invalid ranges.
 ‍
 
 [1]: #lesson
+[3]: #link
```

Comment:
Sorry, I didn't mean for that to be taken so literal. The #link in this case would be #inkling-types if you want to link something to the inkling types section of the docs.

Let me know if that makes sense or not.